Firmware/RTK_Surveyor/Buttons.ino

```diff
@@ -31,11 +33,20 @@ void powerDown(bool displayInfo)
     delay(2000);
   }
 
-  pinMode(pin_powerSenseAndControl, OUTPUT);
-  digitalWrite(pin_powerSenseAndControl, LOW);
+  if (pin_powerSenseAndControl >= 0)
+  {
+    pinMode(pin_powerSenseAndControl, OUTPUT);
+    digitalWrite(pin_powerSenseAndControl, LOW);
+  }
+
+  if (pin_powerFastOff >= 0)
+  {
+    pinMode(pin_powerFastOff, OUTPUT);
+    digitalWrite(pin_powerFastOff, LOW);
+  }
 
-  pinMode(pin_powerFastOff, OUTPUT);
-  digitalWrite(pin_powerFastOff, LOW);
+  if ((productVariant == RTK_FACET) || (productVariant == RTK_FACET_LBAND) || (productVariant == REFERENCE_STATION))
+    digitalWrite(pin_peripheralPowerControl, LOW);
```

Comment:
Good catch/add. Tho, I don't think it adds anything. Peripheral power goes down when MOSFET turns off.

Reply:
The issue here is that `pin_powerFastOff` could be undefined. Likewise for `pin_peripheralPowerControl`. You're always setting `pin_powerFastOff` low, even if it's undefined. Just trying to make this a bit more bullet-proof. There are a bunch of new `pin_ = -1` initializers in `RTK_Surveyor.ino` too. Just to identify if the pins are used and have been defined, or not.

Reply:
Yep, I like the -1 and checks. I'm just saying ` digitalWrite(pin_peripheralPowerControl, LOW);` is new, not controlled before. I don't see a problem with it, it's just new.

And yes, controlling undefined pins pin_powerFastOff and pin_powerSenseAndControl was very bad. Good fix!